inttest/calico/calico_test.go

```diff
@@ -72,36 +69,32 @@ func (s *CalicoSuite) TestK0sGetsUp() {
 
 	createdTargetPod, err := kc.CoreV1().Pods("default").Create(s.Context(), &corev1.Pod{
 		TypeMeta:   metav1.TypeMeta{Kind: "Pod", APIVersion: "v1"},
-		ObjectMeta: metav1.ObjectMeta{Name: "nginx"},
+		ObjectMeta: metav1.ObjectMeta{Name: "nginx-worker0"},
 		Spec: corev1.PodSpec{
-			Containers: []corev1.Container{{Name: "nginx", Image: "docker.io/library/nginx:1.23.1-alpine"}},
+			Containers: []corev1.Container{{Name: "nginx-worker0", Image: "docker.io/library/nginx:1.23.1-alpine"}},
```

Comment:
There's another flavor, which seems to be "smaller": `docker.io/library/nginx:1.23.3-alpine-slim` (but that might not include `wget`, I didn't check).